src/usePagenation/createPagenation.ts

```diff
@@ -0,0 +1,85 @@
+
+
+interface pagenation {
+    current: Number;
+    pageSize: Number;
+    total: Number;
+    data: Array<any> | null;
+    onChange: Function;
+    run: Function;
+}
+
+export default class CreatePagenation {
+    current: Number;
+    pageSize: Number;
+    total: Number;
+    request: (params: any) => Promise<any>;
+    options: any;
+    updateFunc: Function;
+    data: Array<any> | null;
+    loading: Boolean;
```

Comment:
You're typing with the `Boolean` constructor not the `boolean` type here.
```suggestion
    loading: boolean;
```